lessons/Algebra2/linear1-exploring-states/langs/en-us/pages/exploring-states.adoc

```diff
@@ -13,7 +23,11 @@ Then, click "Run" and type `states-table` into the Interactions Area on the righ
 |
 |===
 
-@n What code will produce a table showing the number of states in each region? @fitb{}{@ifsoln{@show{(code '(count states-table "region"))}}}
+
+== Digging into the Code
```

Comment:
This set of questions is a great way to keep `sort` and `first-n-row` fresh in students' minds. But is that actually important for Alg2? I'm not opposed to keeping these questions, but it's worth 5m of talking about whether they should go on an optional page instead

Reply:
agreed to remove these questions about building tables.

instead - things like find a state that has a lot of veterans. if you pick two states that have lots of in